examples/main.tf

```diff
@@ -8,12 +8,12 @@ module "delegate" {
   namespace = "harness-delegate-ng"
   manager_endpoint = "PUT_YOUR_MANAGER_URL"
   delegate_image = "PUT_YOUR_DELEGATE_IMAGE"
-  replica = 1
+  replicas = 1
```

Comment:
This is a previously missed mistake. But since this example file is not used, no escalation happened. Good discovered here when I do the testing